packages/suite/src/components/wallet/AmountComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import { isNftTokenTransfer, formatAmount, getTxOperation } from '@suite-common/wallet-utils';
+import { TokenTransfer } from '@trezor/connect';
+
+import { FormattedNftAmount } from 'src/components/suite/FormattedNftAmount';
+import { FormattedCryptoAmount } from 'src/components/suite/FormattedCryptoAmount';
+
+type AmountComponentProps = {
+    transfer: TokenTransfer;
+    useStyled?: boolean;
```

Comment:
What does this mean? I don't understand the naming and think you don't use it at all.

Reply:
And the same for contract - it's also not used

Reply:
Yes, it was forgotten there, will remove it now, thanks